docs/user/trusted-publishers/using-a-publisher.md

```diff
@@ -209,3 +211,184 @@ below describe the setup process for each supported trusted publisher.
+
+=== "Google Cloud"
+
+    You can use the <https://pypi.org/project/id/> tool to automatically detect
+    and produce OIDC credentials on Google Cloud services.
+
+    First, ensure that `id` and `twine` are installed in the environment you
+    plan to publish from:
+
+    ```
+    $ python -m pip install -U id twine
+    ```
+
+    If you're unsure what the email address is for the service account your
+    service is using, you can verify it with:
+
+    ```
+    $ python -m id pypi -d | jq 'select(.email) | .email'
+    ```
+
+    Generate an OIDC token from within the environment and store it. The
+    audience should be either `pypi` or `testpypi` depending on which index you are
+    publishing to:
+
+    ```
+    $ oidc_token=$(python -m id pypi)
+    ```
+
+    !!! note
+        `pypi` is only correct for PyPI. For TestPyPI, the correct
+        audience is `testpypi`. More generally, you can access any instance's expected
+        OIDC audience via the `{index}/_/oidc/audience` endpoint:
+
+        ```console
+        $ curl https://pypi.org/_/oidc/audience
+        {"audience":"pypi"}
+        ```
+
+    Finally, we can submit that token to PyPI and get a short-lived API token
+    back:
+
+    ```bash
+    resp=$(curl -X POST https://pypi.org/_/oidc/mint-token -d "{\"token\": \"${oidc_token}\"}")
+    api_token=$(jq -r '.token' <<< "${resp}")
+    ```
+
+    !!! note
+    
+        This is the URL for PyPI. For TestPyPI, the correct
+        domain should be is `test.pypi.org`.
+
+    This API token can be fed into `twine` or any other uploading client:
+
+    ```bash
+    TWINE_USERNAME=__token__ TWINE_PASSWORD=${api_token} twine upload dist/*
+    ```
+
+=== "ActiveState"
+
+    ActiveState's Platform works as a zero-config CI solution for your dependencies to automatically build cross-platform wheels of your PyPI projects. Once you're set up on the Platform and have linked your PyPI project, you're ready to publish. For more information on getting started with ActiveState, go [here](https://docs.activestate.com/platform/start/pypi). To begin:
+
+    Publish your package to ActiveState's catalog. This will allow ActiveState's Platform to build it for you.
+
+    1. Run the following command using the State Tool CLI:
+        ```
+        state publish --namespace private/ORGNAME --name PKG_NAME PKG_FILENAME --depend "builder/python-module-builder@>=0" --depend "language/python@>=3" --depend "language/python/setuptools@>=43.0.0" --depend "language/python/wheel@>=0"
+        ```
+        Replace the placeholder values in the block above with your ActiveState organization name--this will usually be `USERNAME-org` (ORGNAME), package name (PKG_NAME), and the filename of your sdist or source tarball (PKG_FILENAME) and run the command. Take note of the TIMESTAMP in the output.
+
+        *Note: The namespace must start with `private/` followed by your organization name. You can also append additional 'folder' names if desired.* 
```

Comment:
This `*Note*` does not conform to other updated `!!! note`

Reply:
05ecc46